src/domain/hanke/edit/HankeForm.test.tsx

```diff
@@ -111,9 +110,6 @@ describe('HankeForm', () => {
     await user.click(screen.getByRole('radio', { name: 'Suunnittelu' }));
 
     await user.click(screen.getByRole('checkbox', { name: 'Hanke on YKT-hanke' }));
-
-    await user.click(screen.getByText('Hankkeen suunnitteluvaihe'));
```

Comment:
Looking at this tests name. Maybe the whole test can be removed? Alternatively, if there are valid steps, the name should be modified accordingly.

Reply:
Argh, I missed this because it says 'suunnitteluVaihde' and not 'suunnitteluVaihe'